src/main/scala/inox/solvers/smtlib/SMTLIBParser.scala

```diff
@@ -81,6 +81,10 @@ trait SMTLIBParser {
       case _ => IntegerLiteral(n)
     }
 
+    // FIXME smtlib.theories.BitVectorConstant.unapply is broken! This is the correct implementation (i.e. no toInt)
+    case QualifiedIdentifier(SMTIdentifier(SSymbol(cst), Seq(SNumeral(size))), None) if cst startsWith "bv" =>
+      BVLiteral(BigInt(cst drop 2), size.toInt)
```

Comment:
I'm waiting for scala-smtlib to be available on sonatype to push an update for this.